pages/rs_quality/fair.md

```diff
@@ -1,5 +1,178 @@
+additional comments that explain lines of your code, to **project-level documentation** (including README, LICENCE, CITATION, CONTRIBUTING, etc. files) 
+that help to discover it, explain the legal terms of reusing it, describe its functionality and how to install, run and contribute to it, 
+to whole websites full of documentation with function definitions, usage examples, tutorials and guides. 
+You many not need as much documentation as a large commercial software product, but making your code reusable relies on other people being able to understand 
+what your code does and how to use it.
+
+### Software repositories and registries
+
+Having somewhere to share your code is fundamental to making it findable and accessible. 
+Your institution might have a code repository, your research field may have a practice of sharing code via a specific website, archive or journal,
+or your version control system might include an online component that makes sharing different versions of your code easy. 
+You should check the rules or guidelines of your institution, grant or domain on publishing code, as well as any licenses of the code your software depends on or reuses.
+
+Some examples of commonly used software repositories and registries include:
+
+- general-purporse software repositories - [GitHub][github] and [GitLab][gitlab]
+- programming language-specific software repositories - [PyPi][pypi] (for Python) and [CRAN][cran] (for R)
+- software registries  - [BioTools][biotools] (for biosciences) and [Awesome Research Software Registries][awesome-rs-registries], providing a list of research software registries (by country, organisation, domain and programming language) where research software can be registered to help promote its discovery
+
+### Persistent identifiers
+
+Unique persistent identifiers, such as Digital Object Identifiers (DOIs) provided by Zenodo, FigShare and similar digital archiving services, and commits/tags/releases used by GitHub and similar code sharing platforms, 
+help with findability and accessibility of your software, and can help you get credit for your work by providing citable references.
+
+### Tools for assessing FAIRness of software
+
+Here are some tools that can check your software and provide an assessment or measurement of its FAIRness:
+
+- [FAIRsoft evaluator][fair-rs-evaluator]
+- [FAIR software test][fair-rs-test]
+
+The table below provides a summary of how different tools and practices help with the FAIR software principles.
+
+| Tools and practices                                                                                  | Findable | Accessible | Interoperable | Reusable |
+| ---------------------------------------------------------------------------------------------------- | -------- | ---------- | ------------- | -------- |
+| Virtual development environments                                                                     |          | x          |               | x        |
```

Comment:
Not accessibility, but +1 R3: software meets domain-specific community standards